### ISSUE TITLE
Using \Neos\Flow\Mvc\Dispatcher::emit(After|Before)ControllerInvocation breaks compile time

### ISSUE THREAD

**Issue by fcool** [neos/flow-development-collection#2528]
Together with Flow 6.x the Cli commands have been seperated from the Http ActionControllers. 
After this change the named slots are run during compile time, which is breaking, as the compile time object manager is very limited in its support of building the slot instances.

Flow: 6.3

**Comment by sorenmalling** [comment]
@albe Can this be closed after #2529 is merged? (And why does it not always close ticket after merging?)

**Comment by albe** [comment]
Can be closed and I think it will only auto-close once the change is merged into the default branch. So it works as expected for master-target PRs and oddly for others (see https://stackoverflow.com/questions/12235620/automatically-closing-issue-from-pull-request-in-github). Optimally, the "Fixes/Resolves #XYZ" would also be in the commit message, as at least then the issue will auto-close when doing upmerges.

**Comment by fcool** [comment]
But there IS a "Fixes #2528" in the commit message!

What had to be done different?

**Comment by albe** [comment]
Hmmm, then that should indeed have self-closed on merge to master. Since that didn't happen, I assume GH didn't like the ":" between the "Fixes" and the issue number? https://github.com/neos/flow-development-collection/pull/2529/commits/add3b78597c5a8a090205c4ac8f4352ea9d00d43

It's brittle at best, so I guess we can't fully prevent issues from staying open after merge and occasionally closing them manually. Plus I need to start adding those "Fixes/Resolves #X" commit messages myself. My reason to avoid that was always that the issue number only has meaning inside GH, but the git history lives beyond GH soooo... but I guess I'm fighting ghosts in that case.
